packages/docs/getting-started/running-a-validator.md

```diff
@@ -271,10 +273,16 @@ You can see all the options using the following command:
 ./run-docker-validator-network.sh help
 ```
 
+## Get elected as validator
+
 ### Obtain and lock up some Celo Gold for staking
 
 To participate in The Great Celo Stake Off (aka TGCSO) and get fauceted it's necessary to register online via an [online form](https://docs.google.com/forms/d/e/1FAIpQLSfbn5hTJ4UIWpN92-o2qMTUB0UnrFsL0fm97XqGe4VhhN_r5A/viewform).
 
+### Submitting transactions
+
+We created the `celocli` to allow users to easily interact with the smart contracts on the command line. The `celocli` generally expects a celo node, by default we are using the RPC API interface at `http://localhost:8545`. To submit transactions, the node should have the account unlocked (the one that is usually specified with `--from`). You can specify that through the `--unlock` parameter when starting your celo node. Otherwise you can use `celocli` to unlock the account with `celocli account:unlock` (assuming you have the `personal` module enabled).
```

Comment:
`to easily interact with the core Celo smart contracts via the command line`.

`The `celocli` generally expects a Celo node that it can send RPCs to. By default, it uses the RPC API interface at `http://localhost:8545`, but this can be changed via the `celocli config:set`.

`To sign transactions, this node will need to have access to an unlocked key.`

`your Celo node.`

Remove: `(assuming you have the `personal` module enabled)`

Reply:
Will do with the improved validator setup docs PR